### PR TITLE
add zendesk

### PR DESCRIPTION
**Pull request by PeerRich**
replace intercom with zendesk

once merged, remove intercom from vercel .env

### CODE REVIEW

File: apps/web/components/Shell.tsx [hunk 1]
```diff
@@ -19,7 +19,8 @@ import { Toaster } from "react-hot-toast";
 
 import LicenseBanner from "@ee/components/LicenseBanner";
 import TrialBanner from "@ee/components/TrialBanner";
-import HelpMenuItemDynamic from "@ee/lib/intercom/HelpMenuItemDynamic";
+import IntercomMenuItem from "@ee/lib/intercom/IntercomMenuItem";
```
Comment (PeerRich) [inline apps/web/components/Shell.tsx:22]: renamed for clarity

File: apps/web/components/Shell.tsx [hunk 2]
```diff
@@ -459,7 +460,7 @@ function UserDropdown({ small }: { small?: boolean }) {
               viewBox="0 0 2447.6 2452.5"
               className={classNames(
                 "text-gray-500 group-hover:text-gray-700",
-                "mt-0.5 h-4 w-4 flex-shrink-0 ltr:mr-2 rtl:ml-2"
```
Comment (PeerRich) [inline apps/web/components/Shell.tsx:462]: this was bugging me, could be in a different PR but didnt care

File: apps/web/ee/lib/intercom/IntercomMenuItem.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import { ChatAltIcon } from "@heroicons/react/solid";
```
Comment (PeerRich) [inline apps/web/ee/lib/intercom/IntercomMenuItem.tsx:1]: renamed

File: apps/web/ee/lib/zendesk/ZendeskMenuItem.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+import { useState } from "react";
+
+import ZendeskProvider from "@ee/lib/zendesk/ZendeskProvider";
+
+import classNames from "@lib/classNames";
+import { useLocale } from "@lib/hooks/useLocale";
+
+import { DropdownMenuItem } from "@components/ui/Dropdown";
+
+export default function ZendeskMenuItem() {
```
Comment (PeerRich) [inline apps/web/ee/lib/zendesk/ZendeskMenuItem.tsx:12]: we could DRY this component but i dont think its necessary for now

File: apps/web/.env.example [hunk 5]
```diff
@@ -99,3 +99,6 @@ CALENDSO_ENCRYPTION_KEY=
 
 # Intercom Config
 NEXT_PUBLIC_INTERCOM_APP_ID=
+
+# Zendesk Config
+ZENDESK_KEY=
```
Comment (sean-brydon) [inline apps/web/.env.example:104]: Should this not be `NEXT_PUBLIC_ZENDESK_KEY`

Comment (PeerRich) [reply apps/web/.env.example:104]: yes

**Review by sean-brydon — APPROVED** [review]
LGTM - pulled down and seems to be working as intended

File: apps/web/ee/lib/intercom/HelpMenuItemDynamic.tsx [hunk 6]
```diff
@@ -1,8 +0,0 @@
-import dynamic from "next/dynamic";
-import { Fragment } from "react";
-
-const HelpMenuItemDynamic = process.env.NEXT_PUBLIC_INTERCOM_APP_ID
-  ? dynamic(() => import("./HelpMenuItem"))
-  : Fragment;
```
Comment (zomars) [inline apps/web/ee/lib/intercom/HelpMenuItemDynamic.tsx:6]: The purpose of this is to prevent loading extra JS for self-hosted who don't use Intercom.

Comment (zomars) [reply apps/web/ee/lib/intercom/HelpMenuItemDynamic.tsx:6]: We could do the same with Zendesk
